code/modules/mob/living/carbon/xenomorph/abilities/general_powers.dm

```diff
@@ -902,8 +902,14 @@
 
 	target.last_damage_data = create_cause_data(initial(stabbing_xeno.caste_type), stabbing_xeno)
 
-	stabbing_xeno.visible_message(SPAN_XENOWARNING("\The [stabbing_xeno] skewers [target] through the [limb ? limb.display_name : "chest"] with its razor sharp tail!"), SPAN_XENOWARNING("You skewer [target] through the [limb? limb.display_name : "chest"] with your razor sharp tail!"))
-	playsound(target, "alien_bite", 50, TRUE)
+	if(blunt_stab)
+		stabbing_xeno.visible_message(SPAN_XENOWARNING("\The [stabbing_xeno] spins its tail into [target]'s [limb ? limb.display_name : "chest"], thrashing it!"), SPAN_XENOWARNING("You spin your tail into [target]'s [limb? limb.display_name : "chest"], thrashing it!"))
```

Comment:
thrashing is an odd word to use here, pick a better one

Reply:
![image](https://user-images.githubusercontent.com/53100513/196062955-d0a2fd22-9dde-44d9-a44c-15690a128c4f.png)
reeeeeeee

Reply:
removed